OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -3368,6 +3368,33 @@ public void tagsAvailable(JSONObject tags) {
       assertTrue(second.executed);
    }
 
+   @Test
+   public void testNestedGetTags() throws Exception {
+
+      // Validates that nested getTags calls won't throw a ConcurrentModificationException
+      class DebugGetTagsHandler implements OneSignal.GetTagsHandler {
+         boolean executed = false;
+
+         @Override
+         public void tagsAvailable(JSONObject tags) {
+
+            OneSignal.getTags(new OneSignal.GetTagsHandler() {
+               @Override
+               public void tagsAvailable(JSONObject tags) {
+
```

Comment:
Hmm this test did not fail. Let's make sure both inner and outer `tagsAvailable` callbacks fire